### PR TITLE
formmodal1

### PR DESCRIPTION
**Pull request by sudeakinci**

### CODE REVIEW

File: editor-components/modal/formmodal1/formmodal1.module.scss [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                  transition: 0.5s;
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+
```
Comment (dogusmiuw) [inline editor-components/modal/formmodal1/formmodal1.module.scss:125]: Add media queries.
Same comment for notificationModal2. Apply the same.

File: editor-components/modal/formmodal1/formmodal1.tsx [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+            value: "Talk to us",
+        });
+        this.addProp({
+            type: "string",
+            key: "context",
+            displayer: "Context",
+            value: "Need a demo? or help with anything? Get in touch with our amazing team of experts at your service.",
+        });
+        this.addProp({
+          type: "array",
```
Comment (dogusmiuw) [inline editor-components/modal/formmodal1/formmodal1.tsx:30]: Create a type for this prop.

File: editor-components/modal/formmodal1/formmodal1.tsx [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+          key: "input-items",
+          displayer: "Input Items",
+          value: [
+            {
+              type: "object",
+              key: "input-item",
```
Comment (dogusmiuw) [inline editor-components/modal/formmodal1/formmodal1.tsx:36]: Use camelCase for all keys.

File: editor-components/modal/formmodal1/formmodal1.tsx [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+            });
+        });
+        return value;
+      }
+    
+      return(
+        <div className={this.decorateCSS("container")}>
+          <div className={this.decorateCSS("max-content")}>
+            <div className={this.decorateCSS("page")}>
+              <div className={this.decorateCSS("left-page")}>
```
Comment (dogusmiuw) [inline editor-components/modal/formmodal1/formmodal1.tsx:279]: If there is no image uploaded, delete img's div. Make the other divs strech for available space.

**Review by dogusmiuw — CHANGES_REQUESTED** [review]
Please handle the changes.

**Comment by dogusmiuw** [comment]
please exclude registerables.tsx file from the pr and send again

**Review by dogusmiuw — APPROVED** [review]

**Review by Ayazelar — CHANGES_REQUESTED** [review]
.

File: editor-components/modal/formmodal1/formmodal1.tsx [hunk 5]
```diff
@@ -0,0 +1,543 @@
...
+
+    return (
+
+      <div className={this.decorateCSS("container")}>
+        <div className={this.decorateCSS("max-content")}>
+          <div className={`${this.decorateCSS("page")} ${!imageVal && this.decorateCSS("single-page")}`}>
+            <div className={this.decorateCSS("exit-icon")}>
+              <ComposerModalClose>
+                <ComposerIcon
+                  propsIcon={{ className: this.decorateCSS("exit-icon"), style: { width: "25px", height: "25px" } }}
```
Comment (Ayazelar) [inline editor-components/modal/formmodal1/formmodal1.tsx:438]: remove style and give that values in CSS file

**Review by Ayazelar — CHANGES_REQUESTED** [review]
.

**Review by Ayazelar — APPROVED** [review]
.

**Pull request merged**